docs/site/Decorators.md

```diff
@@ -132,56 +132,185 @@ accordingly or do a composition of them:
   For more usage, refer to [Routing to Controllers](controllers.htm#routing-to-controllers)
 
 ### Parameter Decorator
+ 
+  Syntax: see [API documentation](https://github.com/strongloop/loopback-next/blob/master/packages/openapi-v3/src/parameter-decorator.ts#L18-L29)
 
-  Syntax: [`@param(paramSpec: ParameterObject)`]()
+  Shortcut Pattern: `@param.${in}.${type}(${name})`
 
-  Optional Pattern: `@param.${in}.${type}(${name})`
-
-  `@param` can be applied to method itself or specific parameters, it describes
-  an input parameter of a Controller method.
+  `@param` is applied to controller method parameters to generate OpenAPI parameter specification for them.
 
   For example:
 
   ```ts
-    // example 1: decorator `@param ` applied on method level
-    class MyController {
-      @get('/')
-      @param(offsetSpec)
-      @param(pageSizeSpec)
-      list(offset?: number, pageSize?: number) {}
-    }
+  import {get, param} from '@loopback/rest';
+
+  const categorySpec = {
+    name: 'category',
+    in: 'path',
+    required: true,
+    schema: {type: 'string'}
+  }
+
+  const pageSizeSpec = {
+    name: 'pageSize',
+    in: 'query',
+    required: false,
+    schema: {type: 'integer', format: 'int32'}
+  }
+
+  class MyController {
+    @get('Pets/{category}')
+    list(
+      @param(categorySpec) category: string,
+      @param(pageSizeSpec) pageSize?: number,
+    ) {}
+  }
   ```
 
-  or
+  Writing the whole parameter specification is tedious, so we've created shortcuts to define
+  the params with the pattern `@param.${in}.${type}(${name})`:
+
+  - in: The parameter location. It can be one of the following values: `query`, `header`, `path`.
+  - type: A [common name of OpenAPI primitive data type](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#data-types). 
+  - name: Name of the parameter. It should be a `string`.
+
+  Take specifying a data from query as an example, a list of available shortcuts for can
```

Comment:
I guess I'm confused as to what this sentence is trying to say. Did you possibly mean to say `Taking` instead of `Take`?